src/Symfony/Bridge/Doctrine/Form/ChoiceList/DoctrineChoiceLoader.php

```diff
@@ -35,8 +35,6 @@ class DoctrineChoiceLoader implements ChoiceLoaderInterface
     private $choiceList;
 
     /**
-     * Creates a new choice loader.
-     *
      * Optionally, an implementation of {@link EntityLoaderInterface} can be
```

Comment:
The problem is that the "Optionally..." part becomes the description, which it is not. That's why we keep the generic part you've removed.

Reply:
Arf, ok. Reverted, thanks!